packages/ubuntu_wsl_setup/lib/main.dart

```diff
@@ -47,7 +47,9 @@ screens, yet allowing user to overwrite any of those during setup.
         variant.value = value;
         if (value == Variant.WSL_SETUP) {
           subiquityMonitor.onStatusChanged.listen((status) {
-            setWindowClosable(status?.state?.isInstalling == false);
+            final closable =
+                status == null || status.state?.isInstalling == false;
```

Comment:
both `status` and `status.state` are nullable. this would still make the window non-closable if `status.state` is null. it would be safer to use `status?.state?.isInstalling != true` because then it wouldn't matter if either of them is null.

Reply:
You're right. Thanks for that!